erp-server/src/main/resources/application.properties

```diff
@@ -5,12 +5,20 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.datasource.url=${SPRING_DATASOURCE_URL}?&rewriteBatchedStatements=true
 spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
 spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+spring.data.rest.basePath=/api
+
 
 # Batching - https://vladmihalcea.com/how-to-batch-insert-and-update-statements-with-hibernate/
-hibernate.jdbc.batch_size=30
-hibernate.order_inserts=true
-hibernate.order_updates=true
-hibernate.jdbc.batch_versioned_data=true
+spring.jpa.properties.hibernate.jdbc.batch_size=30
+spring.jpa.properties.hibernate.order_inserts=true
+spring.jpa.properties.hibernate.order_updates=true
+spring.jpa.properties.hibernate.jdbc.batch_versioned_data=true
+spring.jpa.properties.hibernate.jdbc.use_streams_for_binary=false
+spring.jpa.properties.hibernate.use_streams_for_binary=false
```

Comment:
These binary types are ok for now. Since signatures will get stored as files and file names or paths will probably be what's store for the `signature` field in the user_account table, I might remove them shortly though.

Rest of these properties look 👌. And thanks for fixing the ones I messed up!